ca/ca.go

```diff
@@ -217,6 +230,13 @@ func New(log *log.Logger, db *db.MemoryStore) *CAImpl {
 		log: log,
 		db:  db,
 	}
+
+	oscpRespURL := os.Getenv(ocspResponderURL)
+	if oscpRespURL != "" {
+		ca.ocspRespURL = oscpRespURL
+		ca.log.Printf("Setting OCSP responder URL for issued certificates to %s", ca.ocspRespURL)
```

Comment:
tiny nit: I think using `%q` instead of `%s` will make this output a little bit nicer.

```suggestion
		ca.log.Printf("Setting OCSP responder URL for issued certificates to %q", ca.ocspRespURL)
```

Reply:
Oh, I was not aware of %q to quote something!